modules/cli/src/main/scala/scala/cli/commands/new/New.scala

```diff
@@ -0,0 +1,57 @@
+package scala.cli.commands.`new`
+
+import caseapp.core.RemainingArgs
+import dependency.*
+
+import scala.build.internal.{Constants, OsLibc, Runner}
+import scala.build.options.{BuildOptions, JavaOptions}
+import scala.build.{Artifacts, Logger, Positioned}
+import scala.cli.commands.ScalaCommand
+import scala.cli.commands.shared.{CoursierOptions, HelpCommandGroup}
+
+object New extends ScalaCommand[NewOptions] {
+  override def group: String = HelpCommandGroup.Main.toString
+
+  override def scalaSpecificationLevel = SpecificationLevel.IMPLEMENTATION
```

Comment:
```suggestion
  override def scalaSpecificationLevel = SpecificationLevel.EXPERIMENTAL
```

Since this command is not part of SIP, we should mark it as experimental.